fieldservice_sale/readme/USAGE.rst

```diff
@@ -4,3 +4,6 @@
 * On a Sale Order Line, select a product configured for field service orders
 * Confirm the Sale Order
 * Field Service orders linked to SO lines are created
+* When a Field Service order is completed, the quantity delivered for its
+linked sale order line will be updated to the quantity ordered, indicating
```

Comment:
Indent 3 spaces to have a RST correct multiline bullet paragraph.

Reply:
Thank you! Fixed and squashed